charts/aad-pod-identity/README.md

```diff
@@ -201,8 +201,10 @@ The following tables list the configurable parameters of the aad-pod-identity ch
 | `mic.syncRetryDuration`                  | Override interval in seconds at which sync loop should periodically check for errors and reconcile                                                                                                                                                                                                                            | If not provided, default value is `3600s`                      |
 | `mic.immutableUserMSIs`                  | List of  user-defined identities that shouldn't be deleted from VM/VMSS.                                                                                                                                                                                                                                                      | If not provided, default value is empty                        |
 | `mic.cloudConfig`                        | The cloud configuration used to authenticate with Azure                                                                                                                                                                                                                                                                       | If not provided, default value is `/etc/kubernetes/azure.json` |
+| `mic.updateUserMSIMaxRetry`                        | The maximum retry of UpdateUserMSI call in case of assignment errors                                                                                                                                                                                                                                                                   | If not provided, default value is `2` |
+| `mic.updateUserMSIRetryInterval`                        | The duration to wait before retrying UpdateUserMSI (batch assigning/un-assigning identity from VM/VMSS) in case of errors                                                                                                                                                                                                                                                                       | If not provided, default value is `1` |
```

Comment:
```suggestion
| `mic.updateUserMSIRetryInterval`                        | The duration to wait before retrying UpdateUserMSI (batch assigning/un-assigning identity from VM/VMSS) in case of errors                                                                                                                                                                                                                                                                       | If not provided, default value is `1s` |
```

Reply:
Updated!